src/_pytest/config/__init__.py

```diff
@@ -898,9 +912,30 @@ def __init__(
 
     @property
     def invocation_dir(self) -> py.path.local:
-        """Backward compatibility."""
+        """The directory from which pytest was invoked.
+
+        Prefer to use :attr:`invocation_params.dir <InvocationParams.dir>`.
+
+        :type: py.path.local
+        """
         return py.path.local(str(self.invocation_params.dir))
 
+    @property
+    def rootdir(self) -> py.path.local:
+        """The path to the :ref:`rootdir <rootdir>`.
+
```

Comment:
If we do document `rootpath`, then we should mention here that users should use `rootpath` instead.

```suggestion

Prefer to use :attr:`rootpath` instead.
```